nrf_security/cmake/psa_crypto_config.cmake

```diff
@@ -244,6 +244,7 @@ kconfig_check_and_set_base_to_one(MBEDTLS_SHA256_C)
 # Legacy configurations for mbed TLS APIs
 kconfig_check_and_set_base_to_one(MBEDTLS_CIPHER_C)
 kconfig_check_and_set_base_to_one(MBEDTLS_PK_C)
+# TODO: Should we have MBEDTLS_PKCS5_C here as well?
```

Comment:
No. PSA scope does not have MBEDTLS_PKCS5_C in its configuration files. This should only be applicable in legacy configurations

Reply:
removed TODO